libs/designer-ui/src/lib/table/index.tsx

```diff
@@ -87,6 +89,8 @@ export const TableEditor: React.FC<TableEditorProps> = ({
             labelId={labelId}
             keyTitle={titles[0]}
             valueTitle={titles[1]}
+            editorKeyType={types[0]}
```

Comment:
DictionaryEditor always has key type as string and shouldn't these be set in the editorOptions?

Reply:
I do default dictionary key type to be string, but doesn't table editor support different types for the key? Looking at the editorOptions, I can see that it has a types array... though given that the 2 cases which we use the table editor are both of type string for the key, I still think we should make this a dynamic type, right?